code/modules/mob/abstract/new_player/new_player.dm

```diff
@@ -44,12 +44,38 @@ INITIALIZE_IMMEDIATE(/mob/abstract/new_player)
 		. += "Time To Start: [SSticker.pregame_timeleft][round_progressing ? "" : " (DELAYED)"]"
 		. += "Players: [length(player_list)] Players Ready: [SSticker.total_players_ready]"
 		if(LAZYLEN(SSticker.ready_player_jobs))
+			. += ""
+			. += ""
+			. += "Group antagonists ready:"
+
+			var/list/ready_special_roles = list()
+
+			//Get the list of all the players, if they are ready, get their special roles (aka antagonists) preferences and count them up in a list
+			for(var/mob/abstract/new_player/player in player_list)
+				if(!player.ready)
+					continue
+				for(var/special_role in player?.client?.prefs?.be_special_role)
+					ready_special_roles[special_role] += 1
+
+			//Get the list of all antagonist types, if they require more than one person to spawn, check that there's at least one candidate and if so list the number of candidates for it
+			for(var/antag_type in all_antag_types)
+				var/datum/antagonist/possible_antag_type = all_antag_types[antag_type]
+
+				if(possible_antag_type.initial_spawn_req > 1)
+					if(ready_special_roles[possible_antag_type.role_type])
+						. += "[possible_antag_type.role_text_plural]: [ready_special_roles[possible_antag_type.role_type]]"
```

Comment:
Since this is a new entry does this not act as a newline? Maybe it'd be better to merge it with the "Group antagonists ready" so it's all on the same line

Reply:
It acts as a newline for some ungodly reason, I have not dug into the ancient code that uses the return after it's generated, but this is how it looks like:
![image](https://github.com/Aurorastation/Aurora.3/assets/65877598/795a5aca-a0b5-4d28-9344-e612a4812f63)
![image](https://github.com/Aurorastation/Aurora.3/assets/65877598/648d88b5-fcc7-4b7e-be99-3657badf6833)

Reply:
I'll testmerge it because I'm not sure that order is very satisfactory